docs/proposals/ose-rebase.md

```diff
@@ -39,11 +45,9 @@ Avoiding those diffs will reduce even further the number of conflicts.
 
 Naming conventions we can use in commit messages in OSE:
 * commits that need to be carried long-term such as branding or release tooling changes will start with [CARRY]
-* commits that are fixing branding or release tooling changes will start with [SQUASH]. The rebase process
```

Comment:
Removed because I don't think automatically squashing is trivial - the initial rebase proposal can assume the git history is as expected: Origin commits + carry commits + tito commit, this should always be the git state of the repo between rebases w/o any manual intervention.